tests/generators/test_huggingface.py

```diff
@@ -121,6 +133,18 @@ def test_model(hf_generator_config):
         assert item is None  # gpt2 is known raise exception returning `None`
 
 
+def test_model_chat(mocker, hf_generator_config):
+    # uses a ~350M model with chat support
+    g = garak.generators.huggingface.Model(
+        "microsoft/DialoGPT-small", config_root=hf_generator_config
+    )
+    mock_format = mocker.patch.object(
+        g, "_format_chat_prompt", wraps=g._format_chat_prompt
+    )
+    g.generate("Hello world!")
+    mock_format.assert_called_once()
```

Comment:
This is a great test for where we are now -- would be nice to have something where we could specifically test the deprefixing but there is some unavoidable stochasticity here that I'm not sure how to account for.

Reply:
Fair point, in the vein of wanting less reliance on actual models I think adding a test in the future that mocks a model response completely by offering a known test result should be a reasonable way to approach adding a `unit` test for validation of `deprefix` function.

Reply:
Nice find re: the model. I wonder if we could use something like the `lipsum` module (which we already depend upon) to mock up a test HF generator that could be used in like Many of our tests. Maybe for jan?